cost/azure/unused_ip_addresses/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v4.6
+
+- Added hash_exclude at policy definition segment
```

Comment:
This should be more human readable and describe what the change actually does to how the policy operates rather than just literally describing what changes were made to the code.